packages/entities/entities-certificates/src/components/CACertificateList.vue

```diff
@@ -27,12 +27,12 @@
       </template>
       <!-- Create action -->
       <template #toolbar-button>
-        <!-- Hide Create button if table is empty -->
-        <Teleport
-          :disabled="!useActionOutside"
-          to="#kong-ui-app-page-header-action-button"
+        <component
+          :is="useActionOutside ? 'Teleport' : 'div'"
+          v-bind="useActionOutside ? { to: '#kong-ui-app-page-header-action-button' } : {}"
```

Comment:
suggestion: this could be simplified a bit, but fine to merge as-is if you'd prefer
```suggestion
          :to="useActionOutside ? '#kong-ui-app-page-header-action-button' : undefined"
```

Reply:
We figured it'd be better to fix this [upstream](https://github.com/vuejs/core/pull/9818). I'll close/merge this PR depending on how the upstream PR goes.

Reply:
Update: I have simplified the `to` prop for all instances. [The vue PR](https://github.com/vuejs/core/pull/9818) has not been reviewed yet, so I'm merging this. If the upstream PR is merged, I'll then revert this change.